pillar.example

```diff
@@ -194,10 +194,6 @@ ssh_config:
 
 
 openssh:
-  # Banner file can be retrieved either by TOFS or by url
-  banner_src: banner_fire
-  # banner_src: salt://ssh/files/banner_src  # <- old style
-
```

Comment:
@sticky-note Did you mean these TOFS indications?  We really don't want to encourage people to use the old method.  We want to direct them to put everything under `openssh:tofs:source_files:...`.  If you meant something else, please feel free to reply.

Reply:
No, I'm ok with that as I know now for what reason.